dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -154,13 +156,26 @@ public class FutureEMBuilder extends CompletableFuture<EntityManagerBuilder> imp
      * @param entityClass entity class.
      */
     @Trivial
-    void add(Class<?> entityClass) {
+    void addEntity(Class<?> entityClass) {
         if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
-            Tr.debug(this, tc, "add: " + entityClass.getName());
+            Tr.debug(this, tc, "FutureEMBuilder@" + Integer.toHexString(hashCode()) + " addEntity: " + entityClass.getName());
```

Comment:
Most of this shouldn't have been needed because the `this` parameter ought to have already been printing the hexadecimal hash code of FutureEMBuilder. It should be appearing in the trace if you do the following:
```suggestion
            Tr.debug(this, tc, "addEntity: " + entityClass.getName());
```